front/pages/w/[wId]/builder/labs/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { Page, PaintIcon } from "@dust-tt/sparkle";
+import type { WorkspaceType } from "@dust-tt/types";
+import type { SubscriptionType } from "@dust-tt/types";
+import type { InferGetServerSidePropsType } from "next";
+import { useRouter } from "next/router";
+
+import AppLayout from "@app/components/sparkle/AppLayout";
+import { subNavigationBuild } from "@app/components/sparkle/navigation";
+import { withDefaultUserAuthRequirements } from "@app/lib/iam/session";
+
+const { GA_TRACKING_ID = "" } = process.env;
+
+export const getServerSideProps = withDefaultUserAuthRequirements<{
+  owner: WorkspaceType;
+  subscription: SubscriptionType;
+  gaTrackingId: string;
+}>(async (_context, auth) => {
+  const owner = auth.workspace();
+  const subscription = auth.subscription();
+  if (!owner || !auth.isAdmin() || !subscription) {
+    return {
+      notFound: true,
+    };
+  }
+
+  return {
+    props: {
+      owner,
+      subscription,
+      gaTrackingId: GA_TRACKING_ID,
+    },
+  };
+});
+
+export default function SolutionsIndex({
```

Comment:
Let's call it Labs... maybe try to add a link to Extract from here and reshape the navigation accordingly?